app/src/main/java/com/sereno/serenoandroidapp/models/apis/openweathermap/currentweatherdata/WeatherData.kt

```diff
@@ -1,6 +1,6 @@
 package com.sereno.serenoandroidapp.models.apis.openweathermap.currentweatherdata
 
-data class WeatherData(
+data class CurrentWeatherResponce(
```

Comment:
```suggestion
data class CurrentWeatherResponse(
```

これで良いんだけど、ファイル名も変更しておいてほしい